### PR TITLE
Feature/monthly core

### PR DESCRIPTION
**Pull request by EmilPPrpic**
### Pull request description

- What is new?
  * Implemented second button on email and decline armory screen.
  
- What has changed?
  * Changed mitems texts and manual triggers

### Checklist
- [ ] I ran this code locally

### CODE REVIEW

File: services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl [hunk 1]
```diff
@@ -18,17 +20,53 @@ import signals.SendMatchesSignal;
...
+        agent: Ava(lunchCycleStage == AvaLunchCycleStage.LUNCH_MAIL_SENDING,
+                   onboardingStage == OnboardingStage.FINISHED,
+                   lunchReminderStage != LunchReminderStage.USER_RESPONDED)
+        employee: EmployeeProfile()
+    then
+        modify(agent) {
+            setLunchCycleStage(AvaLunchCycleStage.FIND_AVAILABILITY),
+            setLunchReminderStage(LunchReminderStage.SECOND_REMINDER)
+        }
+
```
Comment (mmatichr) [inline services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl]: check if function call missing

File: services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl [hunk 2]
```diff
@@ -18,17 +20,53 @@ import signals.SendMatchesSignal;
...
         agent: Ava(lunchCycleStage == AvaLunchCycleStage.LUNCH_MAIL_SENDING,
-                   onboardingStage == OnboardingStage.FINISHED)
+                   onboardingStage in (OnboardingStage.STATS_EMAIL, OnboardingStage.FINISHED))
         eval(evaluateCronExpression("* * 8-17 ? * WED *", ts, "Europe/Zagreb")
-            || (agent.getManualTrigger() == true))
+            || (agent.isManualTrigger() == true))
         employee: EmployeeProfile()
     then
-        modify(agent) {setLunchCycleStage(AvaLunchCycleStage.FIND_AVAILABILITY)}
+        modify(agent) {setLunchCycleStage(AvaLunchCycleStage.FIND_AVAILABILITY),
```
Comment (mmatichr) [inline services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl]: formatting

File: services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl [hunk 3]
```diff
@@ -72,7 +110,37 @@ rule "Confirm days"
...
+
+rule "Finish lunch decline form"
+   when
+        Heartbeat(ts: timestamp) from entry-point "signals"
+        signal: SubmitEvent(getParamAsString("submit") == "finished-lunch-decline-form") from entry-point "signals"
+        agent: Ava(lunchReminderStage != LunchReminderStage.USER_RESPONDED)
+   then
+        for (String paramId : signal.getParams().keySet())
+            if (paramId.startsWith("answer")){
+                modify(agent) { getLunchDeclineReasons().add(signal.getParamAsString(paramId)),
```
Comment (mmatichr) [inline services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl]: formatting

File: services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl [hunk 4]
```diff
@@ -72,7 +110,37 @@ rule "Confirm days"
         agent.confirmingDaysScreen();
         modify(agent) {updateAvailableDays((ArrayList<String>)(signal.getParam("availableDays")))};
         agent.send(CultureMaster.ID, new AvaAvailabilitySignal(new AvaAvailability(agent.getId(), agent.getAvailableDays())));
-        modify(agent) {setLunchCycleStage(AvaLunchCycleStage.ACCEPT_MATCH_INFO)};
+        modify(agent) { setLunchCycleStage(AvaLunchCycleStage.ACCEPT_MATCH_INFO),
```
Comment (mmatichr) [inline services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl]: formatting

File: services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl [hunk 5]
```diff
@@ -18,17 +20,55 @@ import signals.SendMatchesSignal;
 
 import models.AvaLunchCycleStage;
 import models.OnboardingStage;
+import models.LunchReminderStage;
+
+rule "Send first reminder email"
+    when
+        Heartbeat(ts: timestamp) from entry-point "signals"
+        eval(evaluateCronExpression("* * 11 ? * THU *", ts, "Europe/Zagreb"))
```
Comment (mjuraj) [inline services/rule_engine/src/main/resources/rules/Ava/LunchCycleStage.drl]: 11 -> *

+ working hours

**Review by mjuraj — APPROVED** [review]

**Pull request merged**